nailgun/entity_mixins.py

```diff
@@ -1304,3 +1374,26 @@ def search_filter(entities, filters):
                 if getattr(entity, field_name) == field_value
             ]
         return filtered
+
+
+def to_json_serializable(obj):
+    """ Transforms obj into a json serializable object.
+
+    :param obj: entity or any json serializable object
+
+    :return: serializable object
+
+    """
+    if isinstance(obj, Entity):
+        return obj.to_json_dict()
+
+    if isinstance(obj, dict):
```

Comment:
Strange, it seems cherry pick not worked as I expected. Going to check the test file because it was covered.